### ISSUE TITLE
Npgsql 6 is not compatible

### ISSUE THREAD

**Issue by hazzik** [nhibernate/nhibernate-core#2994]
Npgsql 6 has [breaking changes](https://www.npgsql.org/doc/release-notes/6.0.html#timestamp-rationalization-and-improvements) regarding DateTime (timespan/timespantz) handling that renders it incompatible with current version of NHibernate. Currently the only way to make it work is to enable `EnableLegacyTimestampBehavior`:

```csharp
AppContext.SetSwitch("Npgsql.EnableLegacyTimestampBehavior", true);
```

**Comment by 54mu3l** [comment]
Any news on this so far?

**Comment by fredericDelaporte** [comment]
Fixed by #3064 in 5.3.x. [Development builds](https://github.com/nhibernate/nhibernate-core#nightly-development-builds) should be available soon on MyGet. The feature will be in 5.3.13.dev builds, not in 5.4 ones.
